front/src/app/pages/agency-dashboard/AgencyDashboardPage.tsx

```diff
@@ -46,15 +45,10 @@ export const AgencyDashboardPage = ({
             tabId: "dashboard",
             label: "Tableau de bord",
             content: (
-              <>
-                <ManageConventionFormSection
-                  routeNameToRedirectTo={"manageConventionInclusionConnected"}
-                />
-                <MetabaseView
-                  title="Tableau de bord agence"
-                  url={agencyDashboardUrl}
-                />
-              </>
+              <MetabaseView
+                title="Tableau de bord agence"
+                url={agencyDashboardUrl}
+              />
```

Comment:
Je me demande si c'est assez intuitif pour ceux qui auraient l'habitude de passer par le formulaire "Id de la convention".

```suggestion
             Cliquer sur l'identifiant de la convention pour y accéder.
              <MetabaseView
                title="Tableau de bord agence"
                url={agencyDashboardUrl}
              />
```

Reply:
j'ai pris en compte